pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -186,6 +186,8 @@ const (
 	TaskRunReasonsResultsVerificationFailed TaskRunReason = "TaskRunResultsVerificationFailed"
 	// AwaitingTaskRunResults is the reason set when waiting upon `TaskRun` results and signatures to verify
 	AwaitingTaskRunResults TaskRunReason = "AwaitingTaskRunResults"
+	// TaskRunReasonResultLargerThanAllowedLimit is the reason set when one of the results exceeds its maximum allowed limit of 1 KB
+	TaskRunReasonResultLargerThanAllowedLimit TaskRunReason = "TaskRunResultLargerThanAllowedLimit"
```

Comment:
Thanks for the PR @chitrangpatel !

I was wondering if there is any reason this is kept in v1beta1 only? If not I can open up a PR to keep v1 and v1beta1 synced for the reconciler changes.

Reply:
I think thats because the reconciler currently uses v1beta1. This reason is only called there. Its ok to keep them both I sync I suppose. That way, when we update the reconciler to point to v1, it works.